spec/features/clearance/visitor_resets_password_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/features/clearance_helpers'
+
+RSpec.feature 'Visitor resets password' do
+  before { ActionMailer::Base.deliveries.clear }
+
+  around do |example|
+    original_adapter = ActiveJob::Base.queue_adapter
+    ActiveJob::Base.queue_adapter = :inline
+    example.run
+    ActiveJob::Base.queue_adapter = original_adapter
+  end
```

Comment:
Do you have a good idea of what this is doing? In general, if I have generated code that's unfamiliar, I call it out in the PR or ask about it in a channel.

Reply:
Good point. Thanks!